### DISCUSSION TITLE
building a CC2500 module only - layout asked

### DISCUSSION THREAD

**Discussion by iz3man** [pascallanger/DIY-Multiprotocol-TX-Module#732] [General]
Hi

I got a Jumper T pro with internal ELRS, and would like to open it and install a CC2500 module only for my remaining Frsky models. I did the reverse with the Jumper T-lite with 4in1 module where I added an interal ELRS. I don't like the backpacks.

So I got all hardware together, from 328p, CC2500 etc. I found one layout where all 4 modules are connected to the Arduino and a know switches between them. 

As I only got on module: Is there a specific layout how to connect one single module only? Couldn't find any. 

Thanks for you help!

I found this on eastbay-rc, but am not sure if it will work with the multi-module firmware?!

![image](https://user-images.githubusercontent.com/11553674/189488244-a4d4b4ce-5fa4-409c-9651-c3f78dc9856b.png)

 btw: This is the module I want to use:

![image](https://user-images.githubusercontent.com/11553674/189488402-f88b6b43-fb58-46b9-8116-4543b877dacb.png)

**Comment by pascallanger** [comment]
There is nothing special, just connect the CC2500 and ignore the other modules.

**Reply by iz3man** [reply]
Exactly what I wanted to hear. :) Thank you!

So basically this here?

![image](https://user-images.githubusercontent.com/11553674/189488808-66916817-3afd-4995-9cb9-3d2352c19730.png)
